### DISCUSSION TITLE
how to order real-time charts like forex app

### DISCUSSION THREAD

**Discussion by bobwatcherx** [zauberzeug/nicegui#709] [Q&A]
### Question

if I want to make a realtime chart and add bar data every 1 second. Think of it as a real-time goods monitor application. how to make it thanks

**Comment by falkoschindler** [comment]
Hi @bobwatcherx,

We appreciate your enthusiasm in reaching out to us, but it's crucial to provide detailed context for your questions. As this is your 23rd inquiry, we'd like to kindly remind you to include the following when asking for assistance:

A clear explanation of your goal.
An overview of the steps you've already taken or attempted.
Specific points where you are encountering difficulties.
By providing this information, you'll not only enable us to better understand your issue, but also allow us to offer more effective guidance. Our team is dedicated to supporting the community, and we strive to address numerous inquiries across multiple platforms daily. With your cooperation, we can ensure a productive and efficient experience for everyone.

Looking forward to your next post with more context!
